packages/trace-viewer/src/ui/clickPointer.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'react';
+
+declare module 'react' {
+  namespace JSX {
+    interface IntrinsicElements {
+      'x-pw-pointer': JSX.IntrinsicElements['div'];
+    }
+  }
+}
+
+export function ClickPointer({ point }: { point: { x: number; y: number } }) {
+  return (
+    <x-pw-pointer
+      style={{
+        position: 'fixed',
+        backgroundColor: '#f44336',
+        boxShadow: '0px 0px 60px 20px #f44336',
```

Comment:
We use a "warning symbol" already to indicate that position is not 100% correct: https://github.com/microsoft/playwright/blob/main/packages/trace-viewer/src/snapshotRenderer.ts#L372-L382.

Reply:
I saw that, but thought that a shadow communicates the "bluriness" of the pointer better. Now coming back to this after lunch, I think you're right and we should just reuse the pattern we already have :D Gonna change that.

Reply:
done in https://github.com/microsoft/playwright/pull/32514/commits/f0d12bfaa53469ce429524f143a4acd3e021fbda, here's how it looks now: <img width="1249" alt="Screenshot 2024-09-09 at 13 29 14" src="https://github.com/user-attachments/assets/b20c8aea-618d-4931-b9d2-9860f8a2d9c3">